api/v1/mongodbcommunity_types.go

```diff
@@ -455,6 +451,31 @@ func (m MongoDBCommunity) MongoSRVURI() string {
 	return fmt.Sprintf("mongodb+srv://%s.%s.%s", m.ServiceName(), m.Namespace, clusterDomain)
 }
 
+// MongoAuthUserURI returns a mongo uri which can be used to connect to this deployment
+// and includes the authentication data for the user
+func (m MongoDBCommunity) MongoAuthUserURI(user scram.User, password string) string {
+	return fmt.Sprintf("mongodb://%s:%s@%s/%s?ssl=%t",
+		url.QueryEscape(user.Username),
+		url.QueryEscape(password),
+		strings.Join(m.Hosts(), ","),
+		user.Database,
+		m.Spec.Security.TLS.Enabled)
+}
+
+// MongoSRVURI returns a mongo srv uri which can be used to connect to this deployment
```

Comment:
inconsistent function name in this docstring

Reply:
fixed, thanks